custom/icds/location_reassignment/parser.py

```diff
@@ -79,6 +83,11 @@ def _parse_row(self, row, location_type_code):
         if self._invalid_row(operation, old_site_code, new_site_code):
             return
         self._note_transition(operation, location_type_code, new_site_code, old_site_code)
+        self.new_location_details[location_type_code].append({
```

Comment:
namedtuple?

Reply:
i did use that earlier, but then it was not really being useful. These are later passed as arguments to a celery task which coverts it back to a dict, so i thought of using a dict directly instead.